W-11021541 Adding Alt. Text (#74)
mulesoft/docs-anypoint-security: modules/ROOT/pages/index-secrets-manager.adoc

This is the only service that can read the actual secrets an due to its nature, it is not accessible from the public network. 

> image::secrets-manager-diagram.png[Diagram connecting Secrets Manager tp secret group to service provider to the supported Mule platform in Anypoint platform.]

Reviewer: Let's try to simplify even further into something like:

```suggestion
image::secrets-manager-diagram.png[Diagram showing how secrets manager interacts with the secrets provider service and Anypoint Platform services.]
```
